dashboards-observability/server/adaptors/custom_panels/custom_panel_adaptor.ts

```diff
@@ -71,12 +71,20 @@ export class CustomPanelsAdaptor {
         objectType: 'operationalPanel',
         maxItems: 10000,
       });
-      return response.observabilityObjectList.map((panel: any) => ({
-        name: panel.operationalPanel.name,
-        id: panel.objectId,
-        dateCreated: panel.createdTimeMs,
-        dateModified: panel.lastUpdatedTimeMs,
-      }));
+      var panelList: any[] = [];
```

Comment:
use `const`?

Reply:
Good point, I thought you couldn't push to a const array.